.github/workflows/test.yml

```
@@ -0,0 +1,174 @@
+    - name: Run tests with coverage
+      run: npm run test:coverage
+    
+    - name: Upload coverage reports
+      uses: actions/upload-artifact@v4
+      with:
+        name: coverage-reports-${{ matrix.node-version }}
+        path: |
+          coverage/
+          coverage-report.json
+        retention-days: 7
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)